apps/minifront/src/components/tx-details/tx-viewer.tsx

```diff
@@ -48,7 +48,7 @@ export const TxViewer = ({ txInfo }: { txInfo?: TransactionInfo }) => {
 
   // use React-Query to invoke custom hooks that call async translators.
   const { data: receiverView } = useQuery(
-    ['receiverView', txInfo, option],
+    ['receiverView', txInfo?.toJson({ typeRegistry }), option],
```

Comment:
Now that we are saving relay transactions, there is a change of an `Any` field needing to be serialized. For that reason, it needs access to the registry to know how to serialize/deserialize. Hence, why you see `typeRegistry` being added to a number of from/toJson() methods.